allennlp/nn/util.py

```diff
@@ -1260,6 +1260,73 @@ def batched_index_select(
     return selected_targets
 
 
+def batched_span_select(target: torch.Tensor, spans: torch.LongTensor) -> torch.Tensor:
+    """
+    The given ``spans`` of size ``(batch_size, num_spans, 2)`` indexes into the sequence
```

Comment:
Our docs only need markdown formatting now, so you can just use single backticks around code/dimensions